src/main/shadow/cljs/devtools/server/web.clj

```diff
@@ -216,8 +217,8 @@
          [:body
           [:div#app]
           [:script {:src "/cache/workspaces/out/js/test.js"}]
-          [:script "shadow.test.workspaces.init();"]
-          ])})))
+          [:script "shadow.test.workspaces.init();"]])})))
```

Comment:
Is there any way to stop parinfer doing this? I like it better visually in my style.

Reply:
I forget that I have printer turned on and it does it automatically. I guess I need to switch to Paredit.